### DISCUSSION TITLE
bash-completion

### DISCUSSION THREAD

**Discussion by GalGavu** [dustinlyons/nixos-config#62] [Q&A]
Hi.
I can see `bash-completion` in packages but at least for me `complete` command doesn't work. I was trying to add it with `bash-completion` in casks but the it says:
> Error: No casks found for bash-completion.

**Comment by dustinlyons** [comment]
Sorry for the late reply.

It looks like you've added `bash-completion` to the `homebrew` section of the configuration. It's looking for the package in the wrong place. You shouldn't have to do anything as it's part of the template.

What's not working with `complete`?
